package.json

```diff
@@ -80,5 +82,12 @@
   },
   "devDependencies": {
     "web-component-tester": "^6.4.0"
+  },
+  "babel": {
+    "env": {
+      "mocha": {
+        "presets": ["es2015"]
```

Comment:
Is this necessary? We're using node8 and that has native es2015. I think we only need to teach about import/export. You can achieve that with `reify`, see https://github.com/home-assistant/home-assistant-js-websocket/blob/master/test/mocha.opts#L1

Reply:
Nice! Good to know!